src/views/Settings.vue

```diff
@@ -349,44 +355,46 @@ export default {
       } catch (e) {
         this.edgeDeviceError = 'Unavailable. Outdated device?'
       }
-    }
-  },
-  computed: {
-    peerConnectionError: function () {
-      console.log('this.$store.state.pnp.peerConnectionStatus', this.$store.state.pnp.peerConnectionStatus)
-      return this.$store.state.pnp.peerConnectionStatus === PEER_CONNECTION_ERROR
     },
-    ...mapState({
-      peerConnectionStatus: state => state.pnp.peerConnectionStatus,
-      isEdgeConnected: state =>
-        state.pnp.peerConnectionStatus === PEER_CONNECTED,
-      pnp: state => state.pnp,
-      edgePeerId: state => state.pnp.remotePeerId,
-      peerFetch: state => state.pnp.peerFetch,
-      edgeVersion: state => state.edgeDevice.edgeSoftwareVersion
-    }),
-    connectStep: function () {
-      let step = 1
+    handleConnectionStep () {
       switch (this.peerConnectionStatus) {
         case PEER_DISCONNECTED:
+          this.peerConnectionIcon = 'cloud-off-outline'
+          this.peerConnectionText = PEER_DISCONNECTED_TEXT
+          break
         case PEER_CONNECTION_ERROR:
-          step = 1
+          this.connectStep = 1
```

Comment:
@vickywane thanks for making these variable names more self explanatory. Helps with reviews.

Reply:
Alright